docs/pages/versions/unversioned/guides/contributing-to-web.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Contributing to Web
+---
+
+There are a couple different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.
```

Comment:
There are a couple of different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.